### ISSUE TITLE
The doubt about the training time of SharpMask

### ISSUE THREAD

**Issue by dearhoper** [facebookresearch/deepmask#58]
I have finished the training of DeepMask and SharpMask models on the COCO2014 training set. But there is a little doubt about the training time. 
The DeepMask model took about 5 days, the same as the description in the “Learning to Segment Object Candidates”. 
But the SharpMask model took more than 2 weeks (on the Nvidia Tesla K40), much more than 2 days which described in the "Learning to Refine Object Segments":
"We use a learning rate of 1e-3 for training the renement stage, which takes about 2 days to train on an  Nvidia Tesla K40m GPU."
What did I miss? the training options?? I used the default options....
